packages/xo-web/src/xo-app/vm-import/ova/index.js

```diff
@@ -98,9 +98,24 @@ function parseTarHeader(header) {
   if (fileName.length === 0) {
     return null
   }
-  const fileSize = parseInt(
-    Buffer.from(header.slice(124, 124 + 11)).toString('ascii'),
-    8
+  // https://stackoverflow.com/a/2511526/72637
+  const sizeBuffer = Buffer.from(header.slice(124, 124 + 12))
+  let fileSize = 0
+  if (sizeBuffer[0] === 128) {
```

Comment:
IMHO it would be best to use hexadecimal or binary literals when checking bits.

Reply:
I don't like hex numbers :/ I either use decimal or binary for bit masks.

Reply:
https://devdocs.io/javascript/lexical_grammar#Binary if you prefer.